__tests__/app.test.js

```diff
@@ -3,6 +3,9 @@ const app = require("../app.js")
 const request = require("supertest")
 const seed = require("../db/seeds/seed.js")
 const testData = require("../db/data/test-data/index.js")
+const allEndpoints = require("../endpoints.json")
+
+// console.log(allEndpoints)
 
 // console.log(testData)
```

Comment:
Remember to remove any commented out code or console logs. What you submit during a pull request should be your best view of a 'final piece of work', and little details like this should be removed in advance.